libtransmission/tr-udp.cc

```diff
@@ -109,6 +109,8 @@ void event_callback(evutil_socket_t s, [[maybe_unused]] short type, void* vsessi
             return;
         }
 
+        auto const from_str = tr_socket_address::from_sockaddr(from_sa).value_or(tr_socket_address{}).display_name();
```

Comment:
Only needed in error cases, and only if trace logs are enabled, maybe turn into a lambda / helper function (which may exist somewhere already?) for lazy evaluation?

Reply:
dcb75f303d7eb5a8b2942d00bb0845b49e5cee68